src/kirby/components/page/page.component.html

```diff
@@ -5,51 +5,55 @@
         text=""
         [defaultHref]="defaultBackHref"
         icon="assets/kirby/icons/svg/arrow-back.svg"
-        *ngIf="!hideBackButton"
+        [style.visibility]="!hideBackButton ? 'visible' : 'hidden'"
       ></ion-back-button>
     </ion-buttons>
     <ion-title>
-      <div [@visibility]="toolbarTitleVisibility">
+      <div class="hide" [ngClass]="{ 'fade-in': toolbarTitleVisible }">
         <ng-container *ngTemplateOutlet="toolbarTitleTemplate"></ng-container>
       </div>
     </ion-title>
-    <ion-buttons *ngIf="stickyActionsTemplate" slot="primary" #stickyToolbarButtons>
-      <div [@visibility]="toolbarStickyActionsVisibility">
-        <ng-container *ngTemplateOutlet="stickyActionsTemplate"></ng-container>
-      </div>
+    <ion-buttons
+      class="hide"
+      slot="primary"
+      #stickyToolbarButtons
+      [ngClass]="{ 'fade-in': toolbarStickyActionsVisible && stickyActionsTemplate }"
```

Comment:
Consider simplifying for readability:
```suggestion
      [class.fade-in]="toolbarStickyActionsVisible && stickyActionsTemplate"
```